build/action.yaml

```diff
@@ -35,18 +40,49 @@ runs:
       uses: andstor/file-existence-action@v3
       with:
         files: yarn.lock
+    - name: Compute lockfile hash
```

Comment:
Since the code is the same in all the actions, what do you think about extracting the logic into a new `actions-node/install-dependencies` action that these actions call to avoid some DRY?

Reply:
Sure can do